src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -0,0 +1,1053 @@
+          ... on CommerceShip {
+            name
+            addressLine1
+            addressLine2
+            city
+            region
+            country
+            postalCode
+            phoneNumber
+          }
+          ... on CommerceShipArta {
+            name
+            addressLine1
+            addressLine2
+            city
+            region
+            country
+            postalCode
+            phoneNumber
+          }
+        }
+        lineItems {
+          edges {
+            node {
+              artwork {
+                slug
+                processWithArtsyShippingDomestic
+                artsyShippingInternational
+                pickup_available: pickupAvailable
+                onlyShipsDomestically
+                euShippingOrigin
+                shippingCountry
+              }
+              shippingQuoteOptions {
+                edges {
+                  ...ShippingQuotes_shippingQuotes
+                  node {
+                    id
+                    isSelected
+                  }
+                }
+              }
+            }
+          }
+        }
+        ...ArtworkSummaryItem_order
+        ...TransactionDetailsSummaryItem_order
+        ...OrderStepper_order
+      }
+    `,
+    me: graphql`
+      fragment Shipping2_me on Me
+        @argumentDefinitions(
+          first: { type: "Int", defaultValue: 30 }
+          last: { type: "Int" }
+          after: { type: "String" }
+          before: { type: "String" }
+        ) {
+        ...FulfillmentDetailsForm_me
+        ...SavedAddresses2_me
+        name
+        email
+        id
+        location {
+          country
+        }
+        addressConnection(
+          first: $first
+          last: $last
+          before: $before
+          after: $after
+        ) {
+          edges {
+            node {
+              id
+              internalID
+              addressLine1
+              addressLine2
+              addressLine3
+              city
+              country
+              isDefault
+              name
+              phoneNumber
+              postalCode
+              region
+            }
+          }
+        }
+      }
+    `,
+  }
+)
+const ArtaErrorDialogMessage = () => (
+  <>
+    There was a problem getting shipping quotes. <br />
+    Please contact{" "}
+    <RouterLink inline to={`mailto:orders@artsy.net`}>
+      orders@artsy.net
+    </RouterLink>
+    .
+  </>
+)
+
+// Get information from the order and user, savedOrderData (shared, computed context about order)
+// initial values for forms, and the current step in the shipping route
+// values for forms
+export const useLoadOrder = (
+  props: ShippingProps
+): {
+  savedOrderData: ReturnType<typeof useLoadComputedData>
+  initialValues: {
+    fulfillmentDetails: FulfillmentValues
+  }
+  targetStep: ShippingRouteStep
+} => {
+  const orderData = useLoadComputedData(props)
+  const initialValues = getInitialValues(props, orderData)
```

Comment:
This hook (hook so it can take advantage of other hooks) loads all the initial order into a shape that we might decide to change. I tried to split things up logically - savedOrderData ends up in the context for other orders, similar to how we use a lot of helper functions in our [current shippingUtils file](https://github.com/artsy/force/blob/dcc35900f3e9e8fb291cd6779b7f993d689544f8/src/Apps/Order/Utils/shippingUtils.ts). This just makes it a little more clear and lets those values change more predictably when the relay order changes. So this last 300 lines or so are pretty important.